.github/workflows/sync_docs.yaml

```diff
@@ -5,15 +5,14 @@ name: Sync docs from Discourse
 on:
   workflow_dispatch:
   schedule:
-    - cron: 00 00 * * *
+    - cron: '53 0 * * *' # Daily at 00:53 UTC
 
 jobs:
   sync-docs:
     name: Sync docs from Discourse
-    uses: canonical/data-platform-workflows/.github/workflows/_sync_docs.yaml@v20.0.1
-    secrets:
-      discourse-api-user: ${{ secrets.DISCOURSE_API_USERNAME }}
-      discourse-api-key: ${{ secrets.DISCOURSE_API_KEY }}
+    uses: canonical/data-platform-workflows/.github/workflows/sync_docs.yaml@v20.0.1
+    with:
+      reviewers: a-velasco,izmalk
```

Comment:
is there are group/team that you're a part of that can be used instead of individual github usernames? e.g. `data-platform-technical-authors`?

Reply:
Since gh teams are cheap it make more sense (change in teams wont require a PR) to have teams for the subteam with it's technical author (e.g.: SQL devs+Andrea, NoSQL+Vladimir, etc).

Reply:
Option 1: (choose  :+1:): 
 
Add Andreia to SQL and set  `data-platform-sql` here. 
Pros: simple
Cons: all SQL devels will receive reviews for all syncs, including MySQL updates from Postgresql, etc.



Option 2:  (choose :-1: ):
```
data-platform-technical-authors
  - data-platform-technical-authors-sql
    - Andreia
  - data-platform-technical-authors-nosql
    - Vladimir
  - data-platform-technical-authors-bigdata
    - ...
```

Default reviewers in DPW: data-platform-technical-authors
MySQL Repo: `data-platform-technical-authors-sql` + `data-platform-sql`
MongoDB Repo: `data-platform-technical-authors-nosql` + `data-platform-nosql`
...
Complex and over-engineered for now.

Option 3:  (choose :eyes: ):
We will commit data-platform-technical-authors only. 
If you want to review docs => we will add you to the group. Maybe seniors should be there.
I see no point to spam all devels with sync docs reviews. We have tech writers now.

Reply:
1. Vladimir is in charge of `bigdata`. I'm with `sql` and `nosql`.
2. If I have a significant doc change that I want Vladimir (or anyone) to review, I ask them directly. These bot PRs are accumulated, miscellaneous changes in a day that aren't intended for thorough reviews - just a brief look-over to make sure there are no glaring technical mistakes or a suspicious contribution. Developers from the relevant team are critical for this.
3. If this is considered "spam", then we should reduce the frequency of the workflow (e.g. weekly).